homie-controller/src/types.rs

```diff
@@ -75,11 +75,19 @@ impl Display for State {
 /// The data type of a Homie property.
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub enum Datatype {
+    /// A [64-bit signed integer](https://homieiot.github.io/specification/#integer).
     Integer,
+    /// A [64-bit floating-point number](https://homieiot.github.io/specification/#float).
     Float,
+    /// A [boolean value](https://homieiot.github.io/specification/#boolean).
     Boolean,
+    /// A [UTF-8 encoded string](https://homieiot.github.io/specification/#string).
     String,
+    /// An [enum value](https://homieiot.github.io/specification/#enum) from a set of possible
+    /// values specified by the property format.
     Enum,
+    /// An RGB or HSV [color](https://homieiot.github.io/specification/#color), depending on the
+    /// property format.
```

Comment:
(optional) can this link to `Property::format`?

Reply:
Linked to Property::color_format() instead.